proto/messages.proto

```diff
@@ -48,6 +48,25 @@ message FileSystemDeleted {
   string id = 1;
 }
 
+message DeploymentGroup {
+  string name = 1;
```

Comment:
I think we should have IDs on these messages and unify everything by ID, including the delete operation.

Reply:
The reason why we don't do it for secrets and file system is because they are identified by their name but that doesn't need to be the case here.